packages/twenty-website/src/app/user-guide/[slug]/page.tsx

```diff
@@ -1,15 +1,23 @@
+'use client';
+import { use } from 'react';
```

Comment:
You can't do that because you call getPost which uses server-side only features (`fs`).

Reply:
(sorry meant to comment on the line above "use client")